src/main/java/org/jvnet/hudson/update_center/Main.java

```diff
@@ -389,7 +376,8 @@ protected JSONArray buildReleaseHistory(MavenRepository repository) throws Excep
                     o.put("title", title);
                     o.put("gav", h.artifact.groupId+':'+h.artifact.artifactId+':'+h.artifact.version);
                     o.put("timestamp", h.getTimestamp());
-                    o.put("wiki", plugin.getWiki());
+                    o.put("wiki", plugin.getPluginUrl());
+                    o.put("url", "https://plugins.jenkins.io/" + h.artifact.artifactId);
```

Comment:
1539d22e615f5fcca97e17e6ba07fcb98612f876@jenkins-infra/backend-update-center2
I'm not sure this is actually needed (Rather I'm not sure release-history.json is actually used).
This looks a change unrelated to the purpose of this request, but it's OK.